neptunecontrib/api/table.py

```diff
@@ -65,10 +66,48 @@ def log_table(name, table, experiment=None):
     _exp.log_artifact(export_pandas_dataframe(table), "tables/" + name + '.html')
```

Comment:
It would be good to provide 'html' parameter to export_pandas_dataframe and be not depended on default value.

Reply:
changed.